### PR TITLE
Adds SessionStoreProvider to InstagramFactory

### PR DESCRIPTION
**Pull request by traedamatic**
This request creates a new class SessionStoreProvider. It will be used in the getClient method of  InstagramFactory Class. It will solve the issue #27.  I tested it in my current Laravel project. I couldn't figure out how to write custom tests for the getLoginUrl method within this pacakge. Help wanted. :)

### CODE REVIEW

File: src/Providers/SessionStoreProvider.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    /**
+     * Use the register method to register items with the container via the
+     * protected $this->container property or the `getContainer` method
+     * from the ContainerAwareTrait.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        $this->getContainer()->share(DataStoreInterface::class, new SessionStore(App::make('Illuminate\Session\Store')));
```
Comment (GrahamCampbell) [inline src/Providers/SessionStoreProvider.php]: This doesn't look like the correct way to do things?

**Comment by vinkla** [comment]
Thanks for your contribution. Though, we've already something else in the works that will replace this setup.

**Comment by traedamatic** [comment]
Okay great! I found a strange behavior in the base package. (https://github.com/larabros/elogram/issues/42) May that goes in the same direction.

**Comment by vinkla** [comment]
@traedamatic we're currently waiting on an update in the elogram package.

**Pull request closed without merging**